nx/lib/nx/defn/expr.ex

```diff
@@ -366,7 +366,20 @@ defmodule Nx.Defn.Expr do
 
   unary_ops =
     [:exp, :expm1, :log, :log1p, :sigmoid, :cos, :sin, :tan, :cosh, :sinh, :tanh] ++
-      [:acosh, :asinh, :atanh, :sqrt, :rsqrt, :cbrt, :is_nan, :negate, :sign, :abs, :bitwise_not] ++
+      [
+        :acosh,
+        :asinh,
+        :atanh,
+        :sqrt,
+        :rsqrt,
+        :cbrt,
+        :is_nan,
+        :is_infinity,
+        :negate,
+        :sign,
+        :abs,
+        :bitwise_not
+      ] ++
```

Comment:
Pls move `is_nan` and `is_infinity` to a separate line so this doesn't break apart. :)